Formula/git-flow-avh.rb

```diff
@@ -1,7 +1,7 @@
 class GitFlowAvh < Formula
   desc "AVH edition of git-flow"
   homepage "https://github.com/petervanderdoes/gitflow-avh"
-  revision 1
+  revision 2
```

Comment:
Bumping the revision is a little over-the-top for just installing some additional completions. Anyone who wants them that urgently can simply `brew reinstall git-flow-avh` after/if this is merged.